infrastructure/persistence/database/connection.py

```
@@ -254,6 +254,53 @@ def _apply_migration_files(conn: sqlite3.Connection) -> None:
+                notes TEXT NOT NULL DEFAULT '',
+                use_legacy_chat_completions INTEGER NOT NULL DEFAULT 0,
+                sort_order INTEGER NOT NULL DEFAULT 0,
+                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+                updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+            )
+        """)
+        conn.execute("INSERT INTO llm_profiles SELECT * FROM llm_profiles_old")
+        conn.execute("DROP TABLE llm_profiles_old")
+        conn.commit()
+        logger.info("llm_profiles table recreated with updated CHECK constraint.")
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)